scripts/dummy.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
keep in mind thtat native are slower than blubird, could skew results during load testing http://softwareengineering.stackexchange.com/questions/278778/why-are-native-es6-promises-slower-and-more-memory-intensive-than-bluebird